drivers/pinmux/pinmux_stm32.c

```diff
@@ -54,10 +54,10 @@ static int stm32_pin_configure(uint32_t pin, uint32_t func, uint32_t altf)
 
 #ifdef CONFIG_PM_DEVICE_RUNTIME
 	ret = pm_device_get(port_device);
-#endif
 	if (ret != 0) {
 		return ret;
 	}
+#endif
```

Comment:
Fine, but fix should be part of 2nd commit

Reply:
oops! txs!